fs/inode/fs_files.c

```diff
@@ -71,6 +71,11 @@ static int files_extend(FAR struct filelist *list, size_t row)
       return 0;
     }
 
+  if (row * CONFIG_NFILE_DESCRIPTORS_PER_BLOCK > _POSIX_OPEN_MAX)
```

Comment:
```suggestion
  if (row * CONFIG_NFILE_DESCRIPTORS_PER_BLOCK > OPEN_MAX)
```

Reply:
In my opinion, _POSIX_OPEN_MAX here is a consistent behavior.
If you look at sysconf(), you will find that sysconf returns _POSIX_OPEN_MAX, not OPEN_MAX. 
long sysconf(int name)
{
..................
  switch (name)
    {
................
      case _SC_OPEN_MAX:
        return _POSIX_OPEN_MAX;

My point is that this open limit should be the same as what sysconf returns. 
Considering that OPEN_MAX may be different from _POSIX_OPEN_MAX in the future. If you do want to use OPEN_MAX, I think sysconf should be modified too.

Thanks for your review. It would be great if you can reply again.